src/main/resources/config.yml

```diff
@@ -1,5 +1,6 @@
 # The config-version line is immutable. Don't change the value, it will reset itself after every reboot.
```

Comment:
@Aurelien30000 where did the first line go?
That would also be a config version bump, otherwise the user wont get the entry if they update from an older version.

Reply:
Forgot to add it :/